src/main/java/controller/ChessController.java

```diff
@@ -0,0 +1,97 @@
+package controller;
+
+import domain.ChessGame;
+import domain.GameStatus;
+import domain.Position;
+import domain.Team;
+import domain.command.Command;
+import domain.command.ContinueCommand;
+import domain.command.EndCommand;
+import domain.command.MoveCommand;
+import domain.command.StartCommand;
+import domain.command.StatusCommand;
+import domain.piece.Piece;
+import domain.piece.PieceWrapper;
+import java.util.List;
+import java.util.stream.Collectors;
+import view.InputView;
+import view.OutputView;
+
+public class ChessController {
+    public void run() {
+        OutputView.printGuide();
+        Command firstCommand = InputView.readStartPhase();
+        if (isEndCommand(firstCommand)) {
+            return;
+        }
+        ChessGame chessGame = initChessGame(firstCommand);
+        List<PieceWrapper> piecesOnBoard = wrapPieces(chessGame.getPiecesOnBoard());
+        OutputView.printChessBoard(piecesOnBoard);
+
+        while (playTurnAndIsNotEnd(chessGame)) {
+        }
```

Comment:
크흠 일단 메서드명 변경해보라는 리뷰만 드렸었는데 아무리 봐도 어색한 코드네요 ㅋㅋㅋ ㅠㅠ
저도 고민을 좀 해봤는데 아래와 같이 변경해볼 수 있지 않을까 싶어요. 이렇게 수정하면 과도한 책임으로 10줄을 이미 한참 넘은 playTurnAndIsNotEnd 메서드도 좀 더 간략하게 줄일 수 있구요.

```suggestion
        Command command = InputView.readWhilePlaying();
        while (isGamePlayable(chessGame, command)) { // 체스보드에서 king이 잡혔거나 command가 end면 false 반환
                playTurn(chessGame, command);        
        }
```

어떤가요? 수정 가능할까요?

Reply:
최대한 말씀하신 대로 깔끔하게 분리해보려고 했는데
```command 입력받기 -> 계속 진행할 수 있는지 확인 -> 턴 실행``` 의 순서를 지켜야 해서, while 문 안에서 if 분기와 break로 처리하지 않고는 깔끔하게 하기가 쉽지가 않네요..
그래서 일단 playTurn 안에서 커맨드를 입력받을 상황이라면 입력받고 return 하도록 수정해봤습니다!